railties/lib/rails/tasks/statistics.rake

```diff
@@ -26,12 +26,14 @@ STATS_DIRECTORIES ||= [
   %w(Channel\ tests     test/channels),
   %w(Integration\ tests test/integration),
   %w(System\ tests      test/system),
-].collect do |name, dir|
-  [ name, "#{File.dirname(Rake.application.rakefile_location)}/#{dir}" ]
-end.select { |name, dir| File.directory?(dir) }
+]
 
 desc "Report code statistics (KLOCs, etc) from the application or engine"
 task :stats do
   require "rails/code_statistics"
-  CodeStatistics.new(*STATS_DIRECTORIES).to_s
+  CodeStatistics.new(
+    *STATS_DIRECTORIES.collect do |name, dir|
+      [ name, "#{File.dirname(Rake.application.rakefile_location)}/#{dir}" ]
+    end.select { |name, dir| File.directory?(dir) }
+  ).to_s
```

Comment:
@zzak @gregmolnar
Maybe something like:
```suggestion
  stats_directories = STATS_DIRECTORIES.filter_map do |name, dir|
      full_dir = "#{File.dirname(Rake.application.rakefile_location)}/#{dir}"
      [ name, full_dir ] if File.directory?(full_dir)
  end
  CodeStatistics.new(*stats_directories).to_s
```

Reply:
I've changed it like this.

Reply:
Hey Greg, are you sure you pushed your changes?
I'm not seeing a change yet.
@zzak Do you think this is good to go after this change?

Reply:
As far as I see I pushed the changes you recommended: https://github.com/rails/rails/pull/47313/files#diff-5ffd58ec8d829a94efe08af959a510626d4fbee2e26a2d90720132e7d08a4ee7R34
I rebased and force pushed.